index.html

```diff
@@ -1,25 +1,219 @@
+                    </div>
+                    <p class="text">
+                        Describe your responsibilities.  Tum dicere exorsus est cur verear, 
+                        ne ad id omnia referri oporteat, ipsum per se ipsam voluptatem, quia 
+                        consequuntur magni dolores eos, qui blanditiis praesentium voluptatum 
+                        deleniti atque insitam in ea quid est eligendi optio, cumque nihil ut 
+                        ipsi auctori huius disciplinae placet: constituam, quid.
+                    </p>
+                    </div>
+                    <div>
+                    <div class="flex">
+                        <p class="sub-title">Lead Designer — Scrypt, Inc.</p>
+                        <p class="date">Jun. 2014 - Sept. 2017</p>
+                    </div>
+                    <p class="text">
+                        Describe your responsibilities.  Tum dicere exorsus est cur verear, 
+                        ne ad id omnia referri oporteat, ipsum per se ipsam voluptatem, quia 
+                        consequuntur magni dolores eos, qui blanditiis praesentium voluptatum 
+                        deleniti atque insitam in ea quid est eligendi optio, cumque nihil ut 
+                        ipsi auctori huius disciplinae placet: constituam, quid.
+                    </p>
+                    </div>
+                    <div>
+                    <div class="flex">
+                        <p class="sub-title">UI Designer — Pinger Inc.</p>
+                        <p class="date">Jun. 2014 - Sept. 2017</p>
+                    </div>
+                    <p class="text">
+                        Describe your responsibilities.  Tum dicere exorsus est cur verear, 
+                        ne ad id omnia referri oporteat, ipsum per se ipsam voluptatem, quia 
+                        consequuntur magni dolores eos, qui blanditiis praesentium voluptatum 
+                        deleniti atque insitam in ea quid est eligendi optio, cumque nihil ut 
+                        ipsi auctori huius disciplinae placet: constituam, quid.
+                    </p>
+                    </div>
+    
+                </section>
+                <!--End of experience section-->
+    
+                <div>
+                    <hr class="section-divide">
+                </div>
+    
+                <!--Education section-->
+                <section>
+                    <h2 class="title">Education</h2>
+                    <div class="flex">
+                        <p class="sub-title">BFA in Graphic Design — James Madison University</p>
+                        <p class="date">Graduated 2014</p>
+                    </div>
+                </section>
+                <!--End of education section-->
+    
+                <div>
+                    <hr class="section-divide hide">
+                </div>
+            </main>    
+        
+            <div class="sidebar">
```

Comment:
For your sidebar, I recommend using an `aside` element instead of a `div`. Works best for semantics too.